karma.conf.js

```diff
@@ -0,0 +1,73 @@
+/*global module*/
+'use strict';
+
+module.exports = function(config) {
+	config.set({
+
+		// base path that will be used to resolve all patterns (eg. files, exclude)
+		basePath: '',
+
+
+		// frameworks to use
+		// available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+		frameworks: ['mocha', 'browserify'],
+
+
+		// list of files / patterns to load in the browser
+		files: [
+			'http://polyfill.webservices.ft.com/v1/polyfill.js?ua=safari/4',
```

Comment:
Should we mention in a comment that it's for PhantomJS that we load this variant of the polyfill?